components/dialog/src/vwc-dialog.scss

```diff
@@ -71,3 +71,26 @@
 		width: 100%;
 	}
 }
+
+.mdc-dialog {
+	$top-position: --_top-position;
+	:host([topPosition='small']) &,
+	:host([topPosition='medium']) &,
+	:host([topPosition*='large']) & {
+		top: var(#{$top-position});
+		height: auto;
+	}
+
+	:host([topPosition='small']) & {
+		--_top-position: 40px;
+	}
+	:host([topPosition='medium']) & {
+		--_top-position: 80px;
+	}
+	:host([topPosition='large']) & {
+		--_top-position: 120px;
+	}
+	:host([topPosition='xlarge']) & {
+		--_top-position: 160px;
+	}
```

Comment:
why aren't these interpolated from the const? as you defined above - `#{$top-position}`

Reply:
I'm still getting used to writing this this way :). I'll fix it